app/src/main/java/com/cpr3663/cpr_scouting_app/PreMatch.java

```diff
@@ -273,32 +280,35 @@ public void onFocusChange(View view, boolean focus) {
                                 adapter.setDropDownViewResource(R.layout.cpr_spinner_item);
                                 // Apply the adapter to the spinner.
                                 spinner_Team.setAdapter(adapter);
-//                                Globals.CurrentTeamToScout = spinner_Team.getSelectedItemPosition();
+                                Globals.CurrentTeamToScout = spinner_Team.getSelectedItemPosition();
                             }
                         }
                     }
                 }
             }
         });
 
-        spinner_Team.setOnFocusChangeListener(new View.OnFocusChangeListener() {
+        spinner_Team.setOnItemSelectedListener(new AdapterView.OnItemSelectedListener() {
             @Override
-            public void onFocusChange(View view, boolean focus) {
-                if (!focus) {
-                    String TeamToScoutStr = String.valueOf(spinner_Team.getSelectedItem());
-                    if (!TeamToScoutStr.isEmpty()) {
-                        int TeamToScout = Integer.parseInt(TeamToScoutStr);
-                        // Since TeamList always has a NO_TEAM entry at index=0, need to check size-1
-                        if (TeamToScout > 0 && TeamToScout < Globals.TeamList.size()) {
-                            // This will crash the app instead of returning null if you pass it an invalid num
-                            String ScoutingTeamName = Globals.TeamList.get(TeamToScout);
-                            text_TeamName.setText(ScoutingTeamName);
-                        } else {
-                            text_TeamName.setText("");
-                        }
+            public void onItemSelected(AdapterView<?> adapterView, View view, int i, long l) {
+                String TeamToScoutStr = String.valueOf(spinner_Team.getSelectedItem());
```

Comment:
I know this isn't your code, but since there's a built-in, i'd prefer:
spinner_Team.getSelectedItem().toString() 
rather than
String.valueof(...)